### PR TITLE
Tohirun refactored

### PR DESCRIPTION
**Pull request by AbdullohIbnAlisher**
some chnaged was added

### CODE REVIEW

File: WpfApp/MainWindow.xaml.cs [hunk 1]
```diff
@@ -23,6 +23,8 @@ public partial class MainWindow : Window
         public MainWindow()
         {
             InitializeComponent();
+//khbcbshksch
+//// added some comment
         }
```
Comment (TohirjonIbnAlisher) [inline WpfApp/MainWindow.xaml.cs:28]: add some commnet

**Review by TohirjonIbnAlisher — CHANGES_REQUESTED** [review]
chnage code

**Comment by AbdullohIbnAlisher** [comment]
ok i chanf=ged

**Review by TohirjonIbnAlisher — APPROVED** [review]
ok

**Pull request merged**